src/Microsoft.Identity.Web.TokenAcquisition/AspNetCore/TokenAcquisition-AspnetCore.cs

```diff
@@ -142,17 +140,9 @@ internal class TokenAcquisitionAspNetCore : TokenAcquisition, ITokenAcquisition,
         public async Task AddAccountToCacheFromAuthorizationCodeAsync(
             AuthorizationCodeReceivedContext context,
             IEnumerable<string> scopes,
-            string authenticationScheme /*= OpenIdConnectDefaults.AuthenticationScheme*/)
+            string authenticationScheme = OpenIdConnectDefaults.AuthenticationScheme /*= OpenIdConnectDefaults.AuthenticationScheme*/)
```

Comment:
I'm not sure about this one. Let's just take a note shouldn't prevent you from merging

Reply:
The interface defines the default.